tgui/packages/tgui/interfaces/CloningConsole.js

```diff
@@ -96,7 +98,7 @@ export const CloningConsole = (props, context) => {
                             <Button
                               content="Save to Disk"
                               icon="upload"
-                              disabled={diskData.length === 0}
+                              disabled={diskData === null}
```

Comment:
This comparison isn't correct; `diskData` is never null, it is always at least an empty list as declared on line 12; perhaps you meant to use `!diskette`?